projects/systelab-components/src/lib/listbox/abstract-listbox.component.ts

```diff
@@ -307,20 +306,20 @@ export abstract class AbstractListBox<T> implements OnInit {
 	}
 
 	private unselectAllNodes() {
-		if (this.gridOptions && this.gridOptions.api) {
-			this.gridOptions.api.forEachNode(node => {
+		if (this.gridOptions && this.gridApi) {
```

Comment:
not needed to check gridOptions because now we do not acces to gridOptions.api

You can remove the full if and use this.gridApi?.forEachNode  the method only will be executed if this.gridApi is defined